src/Components/Search/NewSearch/Mobile/OverlayBase.tsx

```diff
@@ -11,16 +11,19 @@ export const OverlayBase: FC<OverlayBaseProps> = ({ children, onClose }) => {
       dialogProps={{
         width: "100%",
         height: "100%",
-        background: "black",
+        background: "black100",
         justifyContent: "center",
-        backgroundColor: "white",
+        backgroundColor: "white100",
       }}
     >
       <ModalDialogContent
         onClose={onClose}
         width="100%"
         height="100%"
-        margin={0} // TODO: ignored, fix it in palette
+        m={0}
+        // contentProps={{
```

Comment:
In design, when scrolling pills, there is no offset between screen border and a pill:

![Screenshot 2023-05-15 at 11 17 28](https://github.com/artsy/force/assets/3934579/e09c823b-dabb-407e-8cda-151d57cb77cd)

Current [implementation](https://github.com/artsy/palette/blob/main/packages/palette/src/elements/ModalDialog/ModalDialogContent.tsx#L87) of `ModalDialogContent` adds horizontal padding, it looks like this:

![Screenshot 2023-05-15 at 11 19 29](https://github.com/artsy/force/assets/3934579/5737300a-33fd-4ba4-ae9e-361f63672734)

My idea is to allow passing `px` property to the `ModalDialogContent`, but I hope that it's possible to solve this problem without modifying it. I tried to play with `overflow` property of the pills rails component, but without any results.

@dimatretyak maybe you can suggest something here?

Reply:
and @dzucconi

Reply:
Just use `ModalBase` and build your own. That's what that component is for. This dialog component is for a dialog, which this is not.